docker-compose/run.sh

```diff
@@ -2,10 +2,16 @@
 
 set -eux
 
-worker="worker-ubuntu16-04"
+worker_volume="buildbarn-worker-ubuntu16-04"
+tini_volume="buildbarn-tini"
 
-sudo rm -rf bb "${worker}"
-mkdir -m 0777 "${worker}" "${worker}/build"
-mkdir -m 0700 "${worker}/cache"
+docker volume rm "${worker_volume}" || true
+docker volume rm "${tini_volume}" || true
```

Comment:
That volume is never created though, so wouldn't that script fail on first start, given that `buildbarn-tini` is set to be an external volume?